ansible/master_cfg/letsencrypt.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# TODO: Change the email to something that makes more sense
```

Comment:
I will leave it for @mpreisler to decide - it's indeed slightly more accessible than private email. Or we could spam some public mailing list, as this is certificate for public CI :) [and the email notifications should NOT happen, as I understand it, as we will renew much earlier than notice triggers]